### DISCUSSION TITLE
Error with Peer-Groups in Plugin Netbox_BGP

### DISCUSSION THREAD

**Discussion by jeyriku** [netbox-community/netbox#10522] [Q&A] [answered]
Hi, i'm facing an issue when i want to display the Peer-group list from the BGP plug-in

`Server Error

There was a problem with your request. Please contact an administrator.

The complete exception is provided below:

<class 'django.urls.exceptions.NoReverseMatch'>

Reverse for 'peer_group_list' not found. 'peer_group_list' is not a valid view function or pattern name.

Python version: 3.8.10
NetBox version: 3.3.3`

Anyone of you would be so gentle to advise !
Thanks in advance.
Jeremie

**Comment by candlerb** [comment, marked as answer]
You'd have to ask the plugin author.

Are you talking about this one? https://github.com/k01ek/netbox-bgp - if so, I suggest you raise it as an issue there.

**Reply by jeyriku** [reply]
Thanks a lot for your kind advice !!
